inc/manager.php

```diff
@@ -336,17 +336,26 @@ private function do_url_replacement( $html, $extracted_urls ) {
 		$urls = array_combine( $extracted_urls, $extracted_urls );
 		$urls = array_map(
 			function ( $url ) {
+				$is_relative = strpos( $url, '//' ) === 0;
+				if ( $is_relative ) {
+					$url = substr( $url, strpos( $url, '//' ) );
+				}
+
 				$is_slashed = strpos( $url, '\/' ) !== false;
 				$url        = html_entity_decode( $url );
 				$new_url    = apply_filters( 'optml_content_url', $url );
 
+				if ( $is_relative ) {
+					$new_url = substr( $new_url, strpos( $new_url, '//' ) );
+				}
+
 				return $is_slashed ? addcslashes( $new_url, '/' ) : $new_url;
 			},
 			$urls
 		);
 
 		foreach ( $urls as $origin => $replace ) {
-			$html = preg_replace( '/(?<!\/)' . preg_quote( $origin, '/' ) . '/m', $replace, $html );
+			$html = preg_replace( '/(?<![\/|:])' . preg_quote( $origin, '/' ) . '/m', $replace, $html );
```

Comment:
Changed the regex to not match if preceded by `:` this allows to not duplicate replacements